libmattermost.c

```diff
@@ -1763,7 +1831,12 @@ mm_tooltip_text(PurpleBuddy *buddy, PurpleNotifyUserInfo *user_info, gboolean fu
 	_MM_TOOLTIP_LINE_ADD(buddy,user_info,_("First Name"),"first_name",NULL);
 	_MM_TOOLTIP_LINE_ADD(buddy,user_info,_("Last Name"),"last_name",NULL);
 	_MM_TOOLTIP_LINE_ADD(buddy,user_info,_("Email"),"email",NULL);
+	_MM_TOOLTIP_LINE_ADD(buddy,user_info,_("Position"),"position",NULL);	
+	_MM_TOOLTIP_LINE_ADD(buddy,user_info,_("Locale"),"locale",NULL);	
 
+	gchar *rolelist = mm_purple_flag_to_role(purple_blist_node_get_int(PURPLE_BLIST_NODE(buddy),"roles"));

```

Comment:
mm_purple_flag_to_role can be called with NULL - will return 'channel user' as role - and everybody is channel user I thnk ?